### ISSUE TITLE
Feature Request: Sort by Date Added

### ISSUE THREAD

**Issue by seanbperiod** [blastrock/pkgj#4]
The one annoying thing missing from pkgi is being able to see which new titles have been added to the database. Is it possible to add a "Sort by Date Added" feature?

**Comment by seanbperiod** [comment]
Just bumping this to see if this is possible?

**Comment by blastrock** [comment]
Yup, it is possible. I'm planning on changing the way the list is stored in memory and putting it in a sqlite database, then this feature should become easy to implement.

**Comment by seanbperiod** [comment]
Sweet! Looking forward to it. This will be a killer feature!
